package.json

```diff
@@ -45,7 +45,7 @@
     "glob": "^7.1.2",
     "globby": "^6.1.0",
     "graceful-fs": "^4.1.11",
-    "inquirer": "^3.2.1",
+    "inquirer": "^3.1.0",
```

Comment:
Also, you should be using `yarn`, not npm 5. That's the reason your builds are failing.